src/ruby/yast/profiler.rb

```diff
@@ -0,0 +1,71 @@
+
+require "yast"
+
+module Yast
+  class Profiler
+    class << self
+      include Yast::Logger
+
+      RESULT_PATH = "/var/log/YaST2/profiler_result.txt"
```

Comment:
I do not like asking as it can affect profilling result. Actually it is developer tool, so I don't see as such big problem to profile only with root privileges